engine/servermaster/config.go

```diff
@@ -171,7 +174,11 @@ func GetDefaultMasterConfig() *Config {
 		KeepAliveTTLStr:      defaultKeepAliveTTL,
 		KeepAliveIntervalStr: defaultKeepAliveInterval,
 		RPCTimeoutStr:        defaultRPCTimeout,
-		JobBackoff:           jobop.NewDefaultBackoffConfig(),
+		Storage: resModel.Config{
+			Local: resModel.LocalFileConfig{BaseDir: ""},
```

Comment:
Do all executors share the same local directory?

Reply:
This is not necessary, but in practice it is better to configure it as the same local directory.

Reply:
There is only one storage config in servermaser. Does it mean all executors are forced to use the same config?

Reply:
The local directory needs to be configured directly for each executor.